### PR TITLE
[dashmachine] initial chart release for dashmachine

### PR DESCRIPTION
**Pull request by carpenike**
Signed-off-by: Ryan Holt <ryan@ryanholt.net>

#### Special notes for your reviewer:

New chart for dashmachine (https://github.com/rmountjoy92/DashMachine)

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] Chart Version bumped
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[radarr]`)

### CODE REVIEW

File: charts/dashmachine/templates/ingress.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      {{- end }}
+      secretName: {{ .secretName }}
+  {{- end }}
+{{- end }}
+  rules:
+  {{- range .Values.ingress.hosts }}
+    - host: {{ . | quote }}
+      http:
+        paths:
+	{{- range $ingressPaths }}
```
Comment (onedr0p) [inline charts/dashmachine/templates/ingress.yaml:33]: tab

File: charts/dashmachine/templates/ingress.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+          - path: {{ . }}
+            backend:
+              serviceName: {{ $fullName }}
+              servicePort: http
+	{{- end }}
```
Comment (onedr0p) [inline charts/dashmachine/templates/ingress.yaml:38]: tab

File: charts/dashmachine/README.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+dashmachine
+===========
+DashMachine is another web application bookmark dashboard, with fun features.
+
+Current chart version is `1.0.0`
```
Comment (onedr0p) [inline charts/dashmachine/README.md:5]: You can leave the current chart version out

Comment (carpenike) [reply charts/dashmachine/README.md]: Removed

File: charts/dashmachine/templates/ingress.yaml (hunk 2 above)
Comment (carpenike) [reply charts/dashmachine/templates/ingress.yaml]: replaced with spaces

File: charts/dashmachine/templates/ingress.yaml (hunk 1 above)
Comment (carpenike) [reply charts/dashmachine/templates/ingress.yaml]: replaced with spaces

File: charts/dashmachine/templates/ingress.yaml (hunk 2 above)
Comment (carpenike) [reply charts/dashmachine/templates/ingress.yaml]: > Those things above

I believe they're good to go.

**Comment by Ornias1993** [comment]
@onedr0p Maybe next time verify if an added program isn't vaporware or abandonware like this one?

**Comment by onedr0p** [comment]
5 months ago the last substantial commit was 3 months prior.

**Comment by Ornias1993** [comment]
Shoot, my mistake. >.<

**Pull request merged**